cvtRecordApp/Makefile

```diff
@@ -5,6 +5,8 @@ include $(TOP)/configure/CONFIG_APP
 
 DBDINC = cvtRecord menuCvtMethod menuCvtInitState
 
+USR_CFLAGS += /Od
+
```

Comment:
Is turning off optimisation required for it to work?

Reply:
Removed